Containerfile

```diff
@@ -11,7 +11,10 @@ RUN dnf update -y && \
 	python-flake8 \
 	python-virtualenv \
 	python-pip \
-	git
+	gawk \
```

Comment:
With the switch from podman port and manual parsing to podman-compose port we no longer need to add this dep